jx_sqlite/__init__.py

```diff
@@ -211,11 +211,15 @@ def _get(row):
     return _get
 
 
-def set_column(row, col, child, value):
+def set_column(row, col, name, child, value, header):
     if child == ".":
         row[col] = value
```

Comment:
So, you can see the special case I mentioned:  When `child=="."` we do not need to make a structure for this row element, just assign the primitive value.